src/game.rs

```diff
@@ -64,6 +66,8 @@ impl Board {
     }
 }
 
+// REVIEW: clippy is right, passing a word of raw data is cheaper
+//         then a double-word pointer and then dereferencing it.

```

Comment:
Type of this function was made so it could just be used in `.flat_map` without introducing a closure. Your argument is fair, tough